api/clients/v2/verification/cert_verifier.go

```diff
@@ -30,18 +33,31 @@ type ICertVerifier interface {
 //
 // The cert verifier contract is located at https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/core/EigenDACertVerifier.sol
 type CertVerifier struct {
+	logger logging.Logger
 	// go binding around the EigenDACertVerifier ethereum contract
 	certVerifierCaller *verifierBindings.ContractEigenDACertVerifierCaller
+	ethClient          geth.EthClient
```

Comment:
why not reference `ethClient` by * instead?

Reply:
+1, `geth.EthClient` is a struct not an interface

Reply:
[342f6832](https://github.com/Layr-Labs/eigenda/pull/1204/commits/342f6832de75ed9eebd5635d43f6cd1d170e4a91)